docs/cli-reference.md

```
@@ -32,10 +35,71 @@ The CLI stores configuration in `~/.cloud/config.json`:
+
+# Get results with offset
+cloud instance list --limit 10 --offset 10
+
+# Works with all list commands
+cloud function list --limit 20
+cloud container list --limit 50 --offset 100
+cloud storage list --limit 25
+```
+
+When paginated results are available, the CLI shows:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)